### ISSUE TITLE
Implement the entire kernel

### ISSUE THREAD

**Issue by PatrickvL** [Cxbx-Reloaded/Cxbx-Reloaded#65]
All kernel API's need an implementation that matches the Xbox behavior.

One goal is to update all Nt prefixed functions that are currently implemented, to make use of the Ke and Obj prefixed kernel functions.

**Comment by PatrickvL** [comment]
One part of this is https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/issues/146

**Comment by jthecodebuilder** [comment]
#64 appears to be done

**Comment by CakeLancelot** [comment]
> #64 appears to be done..?
I believe those are only stubs, and still need an implementation.

**Comment by PatrickvL** [comment]
Indeed. Just search for `LOG_UNIMPLEMENTED()` in our kernel sources, and you'll see how many are not even started
